azext_edge/edge/_help.py

```diff
@@ -650,6 +650,67 @@ def load_iotops_help():
         short-summary: Dataflow endpoint management.
     """
 
+    helps[
+        "iot ops dataflow endpoint apply"
+    ] = """
+        type: command
+        short-summary: Create or replace a dataflow endpoint resource.
+        long-summary: |
+          An example of the config file format is as follows:
+
+          {
+            "properties": {
+              "endpointType": "Kafka",
+              "kafkaSettings": {
+                "authentication": {
+                  "method": "SystemAssignedManagedIdentity",
+                  "systemAssignedManagedIdentitySettings": {
+                    "audience": "aio-internal"
+                  }
+                },
+                "batching": {
+                  "latencyMs": 5,
+                  "maxBytes": 1000000,
+                  "maxMessages": 100000,
+                  "mode": "Enabled"
+                },
+                "cloudEventAttributes": "Propagate",
+                "compression": "None",
+                "copyMqttProperties": "Disabled",
+                "host": "*.servicebus.windows.net:9093",
```

Comment:
is this string a valid host as is? If not, then change this to be myservicebus or something to make it more clear that this is a value to fill in

Reply:
will update